packages/contracts/src/deploy-config.ts

```diff
@@ -209,6 +209,16 @@ interface RequiredDeployConfig {
    * L2 : Governor voting period in block.
    */
   l2GovernorVotingPeriodBlocks: number
+
+  /**
+   * The salt to determine GovernanceTokenProxy address on L1 and L2.
+   */
+  governanceTokenProxySalt: string
```

Comment:
Shouldn't the salt be private and managed as an environment variable? It seems like there is room for someone to exploit this salt. For example, using the salt to deploy token contracts for scams on other L2s...

Reply:
If someone exploits this salt and deploy the `Proxy` contract ahead of us, the constructor argument `admin` should be set to ours to deploy to the same address. So they cannot upgrade proxy contract without our admin key. So IMO, it's just ok to use this way to manage the salt, wdyt?

Reply:
Ah I see. It looks ok.